core/src/apps/benfen/sign_message.py

```
@@ -0,0 +1,47 @@
+async def sign_message(
+    ctx: wire.Context, msg: BenfenSignMessage, keychain: Keychain
+) -> BenfenMessageSignature:
+
+    await paths.validate_path(ctx, keychain, msg.address_n)
+
+    node = keychain.derive(msg.address_n)
+    pub_key_bytes = seed.remove_ed25519_prefix(node.public_key())
+    address = benfen_address_from_pubkey(pub_key_bytes)
+    bfc_address = try_convert_to_bfc_address(address)
+    if bfc_address is None:
+        raise wire.DataError("bfc_address is none")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)